Recrutify/Recrutify.Services/Services/CandidateService.cs

```diff
@@ -22,14 +24,16 @@ public class CandidateService : ICandidateService
         private readonly IMapper _mapper;
         private readonly IValidator<ProjectResult> _validator;
         private readonly IUserProvider _userProvider;
+        private readonly IUpdateStatusEventProcessor _updateStatusEvent;
 
-        public CandidateService(ICandidateRepository candidateRepository, IMapper mapper, IValidator<ProjectResult> validator, IProjectService projectService, IUserProvider userProvider)
+        public CandidateService(ICandidateRepository candidateRepository, IMapper mapper, IValidator<ProjectResult> validator, IProjectService projectService, IUserProvider userProvider, IUpdateStatusEventProcessor updateStatusEvent, IStatusChangeEventHandler statusChangeEventHandler)
```

Comment:
remove IStatusChangeEventHandler statusChangeEventHandler
rename updateStatusEvent to updateStatusEventProcessor